examples/typescript-react/readme.md

```diff
@@ -0,0 +1,80 @@
+# TypeScript & React TodoMVC Example
+
+> TypeScript is a language for application-scale JavaScript development. TypeScript is a typed superset of JavaScript that compiles to plain JavaScript. Any browser. Any host. Any OS. Open Source.
+
+> _[TypeScript - typescriptlang.org](http://typescriptlang.org)_
+
+> React is a JavaScript library for creating user interfaces. Its core principles are declarative code, efficiency, and flexibility. Simply specify what your component looks like and React will keep it up-to-date when the underlying data changes.
+
+> _[React - facebook.github.io/react](http://facebook.github.io/react)_
+
+## Learning TypeScript
+
+The [TypeScript website](http://typescriptlang.org) is a great resource for getting started.
+
+Here are some links you may find helpful:
+
+* [Tutorial](http://www.typescriptlang.org/Tutorial)
+* [Code Playground](http://www.typescriptlang.org/Playground)
+* [Documentation](http://typescript.codeplex.com/documentation)
+* [Applications built with TypeScript](http://www.typescriptlang.org/Samples)
+* [Blog](http://blogs.msdn.com/b/typescript)
+* [Source Code](http://typescript.codeplex.com/sourcecontrol/latest#README.txt)
+
+Articles and guides from the community:
+
+* [Thoughts on TypeScript](http://www.nczonline.net/blog/2012/10/04/thoughts-on-typescript)
+* [ScreenCast - Why I Like TypeScript](http://www.leebrimelow.com/why-i-like-typescripts)
+
+Get help from other TypeScript users:
+
+* [TypeScript on StackOverflow](http://stackoverflow.com/questions/tagged/typescript)
+* [Forums](http://typescript.codeplex.com/discussions)
+* [TypeScript on Twitter](http://twitter.com/typescriptlang)
+
+_If you have other helpful links to share, or find any of the links above no longer work, please [let us know](https://github.com/tastejs/todomvc/issues)._
+
+## Learning React
```

Comment:
For React stuff just refer people to the react example readme, no point in maintaining the same thing twice.